Core/Utilities/Graphs/GraphUtilities.py

```diff
@@ -5,6 +5,9 @@
     CMS/Phedex Project by ... <to be added>
 """
 
+__RCSID__ = "$Id$"
```

Comment:
Is this really useful after all ? What do we use it for ?

Reply:
I don't think it's VERY useful anymore, not with git at least.

Reply:
Then how about getting ride of it ? We could do it at the same time as we add the copyright if any ?

Reply:
I think it is quite useful still. In the installed software this placeholder is replaced by the latest commit, its date and author. For debugging and understanding the roots of a possible problem this can be quite handy.

Reply:
Ah ? How about git ? 
There are many places where it is not replaced btw:
```
-bash-4.1$ grep -Rn 'RCSID' /opt/dirac/pro/DIRAC/ | grep -c Id
172
```